lib/std/os.zig

```diff
@@ -6056,30 +6056,39 @@ pub fn sendto(
             .SUCCESS => return @as(usize, @intCast(rc)),
 
             .ACCES => return error.AccessDenied,
+            .ADDRINUSE => return error.AddressInUse,
+            .AFNOSUPPORT => return error.AddressFamilyNotSupported,
             .AGAIN => return error.WouldBlock,
             .ALREADY => return error.FastOpenAlreadyInProgress,
-            .BADF => unreachable, // always a race condition
+            .BADF => unreachable, // maybe a race condition. (AF_UNIX) the sent file descriptor is not open.
+            .CONNREFUSED => return error.ConnectionRefused,
             .CONNRESET => return error.ConnectionResetByPeer,
             .DESTADDRREQ => unreachable, // The socket is not connection-mode, and no peer address is set.
             .FAULT => unreachable, // An invalid user space address was specified for an argument.
+            .HOSTUNREACH => return error.NetworkUnreachable,
             .INTR => continue,
-            .INVAL => return error.UnreachableAddress,
+            .INVAL => return error.UnreachableAddress, // (AF_UNIX) incorrect socket family given, or socket in invalid state
             .ISCONN => unreachable, // connection-mode socket was connected already but a recipient was specified
+            .LOOP => return error.SymLinkLoop,
             .MSGSIZE => return error.MessageTooBig,
+            .NAMETOOLONG => return error.NameTooLong,
+            .NETDOWN => return error.NetworkSubsystemFailed,
+            .NETUNREACH => return error.NetworkUnreachable,
+            .NFILE => return error.ResourceLimitReached, // The system-wide limit on the total number of open files has been reached.
```

Comment:
"The system-wide limit on the total number of open files has been reached." => This reads very verbose and "ResourceLimit" is non-explanatory. Is this used elsewhere with the same non-explanatory error name?

Reply:
Yes. it's in the same file